module/os/linux/zfs/zpl_super.c

```diff
@@ -43,7 +43,13 @@ zpl_inode_alloc(struct super_block *sb)
 	return (ip);
 }
 
-static void
+static void __maybe_unused
+zpl_inode_free(struct inode *ip)
+{
+	zfs_inode_free(ip);
+}
+
+static void __maybe_unused
```

Comment:
`__maybe_unused` shouldn't be needed here or above.  We can use the guard macro's here to build the required functions, or not.